plugin/core/views.py

```diff
@@ -241,12 +251,12 @@ def will_save_wait_until(view: sublime.View, reason: int) -> Request:
     return Request.willSaveWaitUntil(will_save_text_document_params(view, reason))
 
 
-def did_save(view: sublime.View, include_text: bool) -> Notification:
-    return Notification.didSave(did_save_text_document_params(view, include_text))
+def did_save(view: sublime.View, include_text: bool, file_name: Optional[str] = None) -> Notification:
+    return Notification.didSave(did_save_text_document_params(view, include_text, file_name))
 
 
-def did_close(view: sublime.View) -> Notification:
-    return Notification.didClose(did_close_text_document_params(view))
+def did_close(file_name: str) -> Notification:
```

Comment:
The ST API calls it "file_name" but it's actually a file path.
Maybe we should be in line with ST API but the latter makes more sense to me.

Reply:
Let's do that in another PR.